syncer/checkpoint_test.go

```diff
@@ -327,10 +327,14 @@ SHOW MASTER STATUS: /* AFTER CONNECTION POOL ESTABLISHED */
 
 func (s *testCheckpointSuite) testTableCheckPoint(c *C, cp CheckPoint) {
 	var (
-		tctx   = tcontext.Background()
-		schema = "test_db"
-		table  = "test_table"
-		pos1   = mysql.Position{
+		tctx  = tcontext.Background()
+		table = &filter.Table{
+			Schema: "test_db",
+			Name:   "test_table",
+		}
+		schemaName = "test_db"
```

Comment:
duplicate with above lines

Reply:
Because `table.Schema` and `table.Name` is used multiple times in this test case, I used `schemaName` and `tableName`. Is it duplicate?